### PR TITLE
Stabilizing approx percentile array

### PR DESCRIPTION
**Pull request by thatzopoulos**
Workaround for #747, we will need a better workaround for this in the future when we make changes to stability checks

### CODE REVIEW

File: extension/src/stabilization_info.rs [hunk 1]
```diff
@@ -14,6 +14,9 @@ crate::functions_stabilized_at! {
     "1.16.0" => {
         approx_count_distinct(anyelement),
         approx_count_distinct_trans(internal,anyelement),
+        accessorpercentilearray_in(cstring),
+        accessorpercentilearray_out(accessorpercentilearray),
+        arrow_uddsketch_approx_percentile_array(uddsketch,accessorpercentilearray),
```
Comment (WireBaron) [inline extension/src/stabilization_info.rs:19]: This last function shouldn't go here, the entry you added for `STABLE_OPERATORS` should cover it.  Do you get an error without this entry?

**Review by WireBaron — APPROVED** [review]
Address the one comment and this should be good to go.

File: extension/src/stabilization_info.rs (hunk 1 above)
Comment (thatzopoulos) [reply extension/src/stabilization_info.rs:19]: It passes without that entry, I will delete it.

Comment (thatzopoulos) [reply extension/src/stabilization_info.rs:19]: If I delete that line, the test fails with:
```
unexpectedly released features: [
    "function arrow_uddsketch_approx_percentile_array(uddsketch,accessorpercentilearray)",
]
```

**Comment by Smittyvb** [comment]
I just trigged a CLA re-check, this should be able to be merged now.

bors retry

**Pull request merged**